packages/components/socialFeed/NewsFeed/MapComponent/Map.web.tsx

```diff
@@ -0,0 +1,27 @@
+import "./styles.css";
+import "leaflet/dist/leaflet.css";
+import { Icon } from "leaflet";
+import { MapContainer, TileLayer, Marker } from "react-leaflet";
+
+type Tlocation = [number, number];
+
+const MapWeb = ({ locationSelected }: { locationSelected: Tlocation }) => {
+  const customIcon = new Icon({
+    iconUrl: "https://i.ibb.co/gSnJ70P/location.png", //load image from web; not work with local image
```

Comment:
maybe we can just use something we host instead of hotlinking some external site (if we can avoid it)

Reply:
this feature not work with local image, so we have to upload on our server and then use it.